RELEASE_NOTES.md

```diff
@@ -1,5 +1,152 @@
 # Release Notes
 
+## 2.2.0
```

Comment:
Should this be 2.2.0-RC1?

Reply:
Generally we don't separate by RC in the release notes - so once 2.2.0 final comes out there will be on RC1 section, just an overall section for the final 2.2.0 release. As such we generally don't bother adding the RC